services/apps/alcs/src/alcs/application/application-submission/application-submission.service.ts

```diff
@@ -25,23 +25,33 @@ export class ApplicationSubmissionService {
     private applicationSubmissionStatusService: ApplicationSubmissionStatusService,
     @InjectRepository(CovenantTransferee)
     private covenantTransfereeRepository: Repository<CovenantTransferee>,
+    @InjectRepository(ApplicationOwner)
+    private applicationOwnerRepository: Repository<ApplicationOwner>,
   ) {}
 
   async get(fileNumber: string) {
-    return await this.applicationSubmissionRepository.findOneOrFail({
-      where: { fileNumber, isDraft: false },
-      relations: {
-        naruSubtype: true,
-        application: {
-          documents: {
-            document: true,
+    const submission = await this.applicationSubmissionRepository.findOneOrFail(
+      {
+        where: { fileNumber, isDraft: false },
+        relations: {
+          naruSubtype: true,
+          application: {
+            documents: {
+              document: true,
+            },
           },
         },
-        owners: {
-          type: true,
-        },
+      },
+    );
+
+    submission.owners = await this.applicationOwnerRepository.find({
```

Comment:
nitpick: Add a comment mentioning this is done separately for performance reasons